include/ibamr/BrinkmanPenalizationStrategy.h

```diff
@@ -78,11 +78,6 @@ class BrinkmanPenalizationStrategy : public SAMRAI::tbox::Serializable
      */
     virtual void postprocessComputeBrinkmanPenalization(double current_time, double new_time, int num_cycles);
 
-    /*!
-     * \brief Set Brinkman penalization penalty factor.
-     */
-    virtual void setBrinkmanCoefficient(double chi);
```

Comment:
Since this is in the public interface for this strategy class, I'd suggest that we indicate that the interface is deprecated rather than removing it entirely.

Reply:
Instead of `chi`, we now use `penalty_factor` and the function name is  `setBrinkmanPenaltyFactor(double penalty_factor)`.

Reply:
Boyce's comment still holds - other people are using this code so lets not change this without deprecating it first.